### PR TITLE
Run NTPD daily to sync time against network clock

### PR DESCRIPTION
**Pull request by aanon4**
Here's a new NTPD proposal (requires the change in PR https://github.com/aredn/aredn/pull/317)

### CODE REVIEW

**Comment by aanon4** [comment]
https://github.com/aredn/aredn/issues/318

**Comment by ab7pa** [comment]
The first script to use the periodic feature.  Looks good.

File: files/etc/mesh-release [hunk 1]
```diff
@@ -1 +1 @@
-3.1.1TUNNEL
```
Comment (dman776) [inline files/etc/mesh-release:1]: Drop this file

**Review by dman776 — CHANGES_REQUESTED** [review]

File: files/etc/mesh-release (hunk 1 above)
Comment (aanon4) [reply files/etc/mesh-release:1]: Oops. Will fix it later this evening.

Comment (aanon4) [reply files/etc/mesh-release]: Fixed

**Review by dman776 — COMMENTED** [review]
I like it.   @ae6xe any feedback

**Review by dman776 — APPROVED** [review]

**Pull request merged**